### PR TITLE
Dashboard graphs paula

### PR DESCRIPTION
**Pull request by paulauus**
All time stats done and page layout changed. Pointers from huddle taken into account. It's a lot of code, and it's going to keep changing, but for now you saw in our huddle what it looks like on the dashboard.

### CODE REVIEW

File: dashboard/dashboard.py [hunk 1]
```diff
@@ -51,3 +56,86 @@
...
+        'release_price': 'Price',
+        'listing_url': 'Listing URL'  # Ensure listing_url is included for hyperlinks
+    }
+)
+
+# Convert price from pennies to pounds
+table_data['Price'] = (table_data['Price'] / 100).round(2)
+
+# Now use st.dataframe with column_config
+st.dataframe(
```
Comment (Leon-JS02) [inline dashboard/dashboard.py:135]: Nice link format feature

File: dashboard/sl_queries.py [hunk 2]
```diff
@@ -1,3 +1,81 @@
...
+    Retrieves game information for the dashboard table including
+    name, genres, release date, platform, price, and listing URL.
+    '''
+    # Adjust the query based on the NSFW filter
+    query = """
+    SELECT
+        g.game_title AS game_name,
+        STRING_AGG(DISTINCT ge.genre_name, ', ') AS game_genres,
+        CAST(g.release_date AS DATE) AS release_date,
+        p.platform_name AS platform,
```
Comment (Leon-JS02) [inline dashboard/sl_queries.py:37]: 🤯

File: dashboard/sl_queries.py [hunk 3]
```diff
@@ -1,3 +1,81 @@
...
+    LEFT JOIN genre ge ON ga.genre_id = ge.genre_id
+    LEFT JOIN game_listing gl ON g.game_id = gl.game_id
+    LEFT JOIN platform p ON gl.platform_id = p.platform_id
+    WHERE (%s OR g.is_nsfw = FALSE)
+    GROUP BY g.game_title, g.release_date, p.platform_name, gl.release_price, gl.listing_url
+    ORDER BY g.release_date DESC;
+    """
+
+    conn = get_connection()
+    df = pd.read_sql(query, conn, params=(show_nsfw,)) # Checkbox result
```
Comment (Leon-JS02) [inline dashboard/sl_queries.py:51]: I fear we have been instructed to not use read_sql for efficiency purposes

Comment (paulauus) [reply dashboard/sl_queries.py:51]: Bad habits die hard.

File: dashboard/dashboard.py
Comment (mehrdadhalali) [inline dashboard/dashboard.py:1]: Could have a different script for drawing the graphs with altair, perhaps?

**Review by mehrdadhalali — APPROVED** [review]

**Pull request merged**